.github/CODEOWNERS

```diff
@@ -200,6 +184,11 @@
 /src/plugins/status_page/ @elastic/kibana-core
 /src/plugins/saved_objects_management/ @elastic/kibana-core
 /src/dev/run_check_published_api_changes.ts @elastic/kibana-core
+/src/plugins/home/public @elastic/kibana-core
+/src/plugins/home/server/*.ts @elastic/kibana-core
+/src/plugins/home/server/services/ @elastic/kibana-core
+/src/plugins/kibana_overview/ @elastic/kibana-core
```

Comment:
I thought Kibana App owned this now, but I could be wrong?

Reply:
@timroes what do you think?

Reply:
Same here as below, we did never create or owned this, as it was created as part of the getting started experience, and from my understanding (and the wiki page) should be Design > Kibana Design. But I would say since this is the "analytics home" page not the general one, we could def argue if this should be owned by someone within the analytics area, in which case I feel/fear it will be Kibana App :D